packages/debugger/lib/session/index.js

```diff
@@ -250,15 +250,17 @@ export default class Session {
             for (let index in generatedSources) {
               index = Number(index); //it comes out as a string due to in, so let's fix that
               const source = generatedSources[index];
-              sources.internal[contextHash][index] = {
-                ...source,
-                compiler: source.compiler || compiler,
-                compilationId: compilation.id,
-                index,
-                id: makeSourceId(compilation.id, contextHash, index),
-                internal: true,
-                internalFor: contextHash
-              };
+              if (source) {
```

Comment:
Maybe add a comment on this line like `//VSCode extension breaks w/o this check` or something?  Otherwise this seems like exactly the sort of thing a well-meaning simplifier might remove later.